### DISCUSSION TITLE
Very interested in your Plugin, How do I install it in my HOOBS box?

### DISCUSSION THREAD

**Discussion by Hedrakk** [richierockskool/homebridge-inkbird-wifi-gateway#3] [Q&A]
Hi,

I'm kind of a noob in all this, but I was wondering how can I install your plugin in my HOOBS Box? I usually search for plugins in the HOOBS library but I can't find it.

I've been searching the web for a good 2 weeks to find out a way to include my Pool Thermometer (IBS-P01B) with the help of the InkBird Wi-Fi Gateway (IBS-M1S) without any hope until I came across your github page.

Thank you!

**Comment by richierockskool** [comment]
I answered you on Stacks, give a little more time and I will update you. Thank you for your interest
